bodhi/server/scripts/manage_releases.py

```diff
@@ -37,7 +37,11 @@
 
```

Comment:
Let's find a way to share code between this old module and the new one. Ideally we leave almost all the code in the new one above and have this module only print the deprecation warning and otherwise use the code from the other module.